collector/metrics.go

```diff
@@ -29,12 +29,16 @@ type Exporter struct {
 
 	labels config.Labels
 	up     *prometheus.GaugeVec
+
+	svcTimes bool
```

Comment:
You can make it public ie. `ExtractServiceTimes` and instead of passing it  as func args set it directly on the metrics sturct. and the default being True.